src/course-outline/CourseOutline.jsx

```diff
@@ -117,6 +121,22 @@ const CourseOutline = ({ courseId }) => {
     errors,
   } = useCourseOutline({ courseId });
 
+  // Use `setToastMessage` to show the toast.
+  const [toastMessage, setToastMessage] = useState(/** @type{null|string} */ (null));
+
+  useEffect(() => {
+    if (location.hash === '#export') {
+      setToastMessage(intl.formatMessage(messages.exportTagsToastMessage));
+      getTagsExportFile(courseId, courseName).finally(() => {
+        setToastMessage(null);
```

Comment:
@pomegranited @bradenmacdonald By default the toast has a delay of 5 seconds before closing. I left the default value and I added success/error toast messages when the export is done; to avoid to close a toast quickly

Commit: https://github.com/openedx/frontend-app-course-authoring/pull/995/commits/449251bbbbdccd432d219f7b697dd1dc884fd0e3